x-pack/plugins/ml/server/routes/schemas/anomaly_detectors_schema.ts

```diff
@@ -105,3 +107,6 @@ export const anomalyDetectionJobSchema = {
   results_retention_days: schema.maybe(schema.number()),
   state: schema.maybe(schema.string()),
 };
+
+const anomalyDetectionJobSchemaObject = schema.object(anomalyDetectionJobSchema);
+export type AnomalyDetectionJob = TypeOf<typeof anomalyDetectionJobSchemaObject>;
```

Comment:
could this be be a `Job` object?
Having a new `AnomalyDetectionJob` type shouldn't be necessary.

Reply:
Changed in https://github.com/elastic/kibana/pull/60376/commits/0ec72638475bfebead195972a3034f6012011104